lib/ardb.rb

```diff
@@ -27,6 +27,9 @@ def self.init(establish_connection = true)
     # setup AR
     ActiveRecord::Base.logger = self.config.logger
     self.adapter.connect_db if establish_connection
+  rescue InvalidDBFileError => exception
+    exception.set_backtrace(caller)
+    raise exception
```

Comment:
@jcredding I'd prefer just putting this rescue on the `require_db_file` line using a `begin...rescue`.  I don't really prefer sweeping rescues like this.